### PR TITLE
Enzyme and Jest Set up

### PR DESCRIPTION
**Pull request by aldy97**
# Changes
- Added some easy-to-start front-end tests
- Set up config for Enzyme and Jest

## Motivation and Context
- We need to add tests into our CI workflow

## Screenshots (if appropriate):
image

## Related Issues:
- In each test file, I need to manually add enzyme config, which is tedious.

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -45,10 +45,25 @@
       "last 1 safari version"
     ]
   },
+  "babel": {
+    "presets": [
+      "react-app"
+    ]
+  },
```
Comment (johnagl) [inline client/package.json:52]: Do we need babel to run Jest/Enzyme?

Comment (aldy97) [reply client/package.json:52]: Yeah we do

**Review by johnagl — APPROVED** [review]
There's a bunch of stuff we don't need to add to package.json such as the following: 
1) babel config with presets react-app
2) jest config with setupFilesAfterEnv, testPathIgnorePatterns
3) jest package
 
This is because we used Creact-React-App which automatically comes with Babel, Webpack, and Jest, see https://medium.com/@imrobinkim/babel-and-webpacks-role-in-create-react-app-3a827ad460a2#:~:text=Babel%20and%20Webpack%20work%20under,files%20for%20Webpack%20and%20Babel..  and https://create-react-app.dev/docs/running-tests/
If we didn't use CRA, we would have to install it like you did. 

Anyways, I managed to figure out how to get the tests to work without those packages, but I'll just approve this branch and fix it in main after since it took a number of steps that would take time to write.

**Comment by aldy97** [comment]
@johnagl Thank you!

**Pull request merged**